zigate-flasher.py

```diff
@@ -152,6 +152,9 @@ def req_chip_id():
     pass
 
 
+@Command(0x36, 'B')
```

Comment:
Hi, thanks for your pull request!
Where did you find this 0x36 command? I can't find this in the docs (JN-AN-1003) for JN5168 bootloader anywhere.

Reply:
Since official NXP flash programmer is written in python, I used uncompyle6 to retrieve the original source and find the command and according response